language/documentation/book/src/variables.md

```diff
@@ -651,22 +651,22 @@ y + z
 
 Move's type system will prevent a value from being used after it is moved. This is the same safety check described in [`let` declaration](#let-bindings) that prevents local variables from being used before it is assigned a value.
 
-For more information, see [Move equality](./equality.md).
+<!-- For more information, see [Move equality](./equality.md). -->
```

Comment:
Why? If we don't want this xref, let's just remove it. Or maybe this formatting is something I don't know about??

Reply:
I'll update the comment